extensions/jira/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Jira Changelog
 
-## [The Custom Field can be marked as required] - 2024-05-16
+## [The custom fields can be render as required] - 2024-05-16
```

Comment:
Side note: I was so interested to see how the `npx @raycast/api@latest publish` will teal with `git commit --amend` and rewrite history in general
Pretty cool!